### DISCUSSION TITLE
Draft Tool Flexibity

### DISCUSSION THREAD

**Discussion by davidkinetisense** [110yards/fantasy#83] [Ideas]
In our group we allow 2 keeps (could be more or less) but we want to be able to put them in a pre draft setup where they were picked in the previous year.  Right now the keepers are in the first 2 spots.  We would like to be able to have the ability to put them in 3 and 11 if that is where they were picked in the previous year.

**Comment by mdryden** [comment]
I think I could add a custom draft order without too much trouble.  I'll put it on the list
